acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -135,6 +135,13 @@ def report_query(node)
     assert_allowed
   end
 
+  step 'static file content endpoint' do
+    # We'd actually need to perform a commit and use its code-id in order to
+    # get back a 200, but we know that a 400 means we got past authorization
```

Comment:
To get back a 200, we would also need a valid file path inside of a module (`modules/files/foo/bar` for example).